pkg/crc/machine/machine.go

```diff
@@ -397,6 +397,13 @@ func (client *client) Start(startConfig StartConfig) (*StartResult, error) {
 		return nil, errors.Wrap(err, "Failed to update cluster ID")
 	}
 
+	if client.monitoringEnabled {
+		logging.Info("Enabling cluster monitoring operator...")
+		if err := cluster.StartMonitoring(ocConfig); err != nil {
+			return nil, errors.Wrap(err, "Cannot start monitoring stack")
+		}
+	}
+
```

Comment:
feels weird to do that before ensuring that kubelet is running, and that the apiserver certs are good. I'd move this just before `	logging.Info("Starting OpenShift cluster ... [waiting 3m]")`

Reply:
Yes this is what I though but this is just after an eventual `DeleteOpenShiftAPIServer`. I don't know what is the state of the cluster after this!